.github/workflows/test-experimental-universe.yml

```diff
@@ -0,0 +1,76 @@
+name: "Test Experimental Universe"
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "**.sh"
+      - "**.bash"
+      - "**.go"
+      - "**.cue"
+      - "**.bats"
+      - "Makefile"
+      - "go.mod"
+      - "go.sum"
+      - ".github/workflows/test-experimental-universe.yml"
+      - "!docs/**"
+
+  pull_request:
+    branches: [main]
+    paths:
+      - "**.sh"
+      - "**.bash"
+      - "**.go"
+      - "**.cue"
+      - "**.bats"
+      - "Makefile"
+      - "go.mod"
+      - "go.sum"
+      - ".github/workflows/test-experimental-universe.yml"
+      - "!docs/**"
```

Comment:
I think we should trigger that ci on change in [`docs/tests/**`](https://github.com/dagger/dagger/tree/main/docs/tests) even if they are not running for now.